Sources/BandcampDownloader/Windows/MainWindow.xaml.cs

```diff
@@ -977,46 +913,6 @@ public partial class MainWindow: Window
             Cursor = Cursors.Arrow;
         }
 
-        private void checkBoxResizeCoverArt_CheckedChanged(object sender, RoutedEventArgs e) {
```

Comment:
Why is the `CheckedChanged` event removed?

Reply:
But this event isn't used to save the setting to file, but to enable/disable the `textBoxCoverArtMaxSize` depending on the checked state of the `checkBoxResizeCoverArt`.